modules/carto/src/sources/vector-table-source.ts

```diff
@@ -45,7 +45,7 @@ export const vectorTableSource = async function (
     urlParameters.columns = columns.join(',');
```

Comment:
I wonder if the API should accept columns as JSON also, would be nicer not to special case it